src/main/java/frc/robot/RobotContainer.java

```diff
@@ -404,47 +425,64 @@ public void configureSwerve() {
                     .withVelocityY(
                         -driver.getLeftX() * MaxSpeed) // Drive left with negative X (left)
                     .withRotationalRate(-driver.getRightX() * MaxAngularRate)));
+
+    /* Right stick absolute angle mode on trigger hold,
+    robot adjusts heading to the angle right joystick creates */
     driver
         .rightTrigger()
         .whileTrue(
             drivetrain.applyRequest(
                 () ->
-                    drive
-                        .withVelocityX(-driver.getLeftY() * MaxSpeed) // Drive forward with
-                        // negative Y (forward)
-                        .withVelocityY(
-                            -driver.getLeftX() * MaxSpeed) // Drive left with negative X (left)
-                        .withRotationalRate(
-                            driver.getRightTriggerAxis()
-                                * MaxAngularRate) // Drive counterclockwise with negative X
-                // (left)
-                ));
+                    azi.withTargetDirection(
+                        Rotation2d.fromDegrees(
```

Comment:
use rotation2d math instead (new Rotation2d(x, y))

Reply:
also I think if you dont specify velocity X and Y its not going to allow you to move while pressing right trigger